src/it/java/com/sun/checkstyle/test/chapter6declarations/rule61numberperline/MultipleVariableDeclarationsTest.java

```diff
@@ -0,0 +1,72 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2019 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.sun.checkstyle.test.chapter6declarations.rule61numberperline;
+
+import org.junit.Test;
+
+import com.google.checkstyle.test.base.AbstractModuleTestSupport;
+import com.puppycrawl.tools.checkstyle.api.Configuration;
+import com.puppycrawl.tools.checkstyle.checks.coding.MultipleVariableDeclarationsCheck;
+
+public class MultipleVariableDeclarationsTest extends AbstractModuleTestSupport {
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/sun/checkstyle/test/chapter6declarations/rule61numberperline";
+    }
+
+    @Test
+    public void testMultipleVariableDeclarations() throws Exception {
+        final String msgComma = getCheckMessage(MultipleVariableDeclarationsCheck.class,
+            "multiple.variable.declarations.comma");
+        final String msg = getCheckMessage(MultipleVariableDeclarationsCheck.class,
+            "multiple.variable.declarations");
+
+        final String[] expected = {
+            "5:5: " + msgComma,
```

Comment:
in `it` we do not need lines in expected, we rely on trailing comments in Input.
We even eventually make such tests format in "tests".

But I am ok to  keep this form for now.

Reply:
I think I didn't properly understand what you try to say.
Besides that, except for the package names, this test is a 1:1 copy from the Google IT for the "same" rule there.

Reply:
yes, my bad, I forgot that we need to do precise validation of column (IDEs and auto-fix tools rely on this) and message(and some Checks have few messages), so such expected data is still required.